pyproject.toml

```diff
@@ -21,12 +20,13 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.7,<3.11"
 structlog = "^21.2.0"
 PyYAML = "^6.0.0"
 pydantic = "^1.9.0"
 typer = "^0.6.1"
 devtools = "^0.9.0"
+"meltano.edk" = {git = "https://github.com/meltano/edk.git", rev = "3-chore-transfer-edk-from-airflow-extension"}
```

Comment:
Temporarily targeting this branch until that branch actually merges over on the github.com/meltano/edk side (https://github.com/meltano/edk/pull/7)

Reply:
```suggestion
"meltano.edk" = {git = "https://github.com/meltano/edk.git"}
```